core/src/com/cloud/agent/api/routing/NetworkElementCommand.java

```diff
@@ -46,6 +47,18 @@ protected NetworkElementCommand() {
         super();
     }
 
+    public void setAccessDetail(final Map<String, String> details) {
+        if (details == null) {
+            return;
+        }
+        for (final Map.Entry<String, String> detail : details.entrySet()) {
```

Comment:
@DaanHoogland sorry don't understand the comment, the idea was to provide/set a detail by passing a map here, which may override a detail (if so asked by the developer)